<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel</class>
 <widget class="QWidget" name="panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="settingGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>200</width>
     <height>270</height>
    </rect>
   </property>
   <property name="title">
    <string>setting</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="runGroup">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>85</y>
     <width>175</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>run</string>
   </property>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>15</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloudButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>cloud</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="loaderGroup">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>10</y>
     <width>185</width>
     <height>270</height>
    </rect>
   </property>
   <property name="title">
    <string>loader</string>
   </property>
   <widget class="QLabel" name="QMLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>222</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Q Matrix:</string>
    </property>
   </widget>
   <widget class="QLabel" name="QFileShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>234</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="QFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="listMode">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ListFile</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dirMode">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ImageFile</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="index">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>180</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AlgorithmsGroup">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>175</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithms</string>
   </property>
   <widget class="QRadioButton" name="SGBMButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SGBM</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ELASButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ELAS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BMButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BM</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="saveGroup">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>175</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>save</string>
   </property>
   <widget class="QPushButton" name="cloudPathButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>87</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
   </widget>
   <widget class="QLabel" name="cloudPath">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>91</y>
      <width>145</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="cloudLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>79</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>cloud path:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="savePathButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
   </widget>
   <widget class="QLabel" name="spShow">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>57</y>
      <width>145</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="spLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>42</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>save path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
    </property>
   </widget>
   <widget class="QCheckBox" name="isSaveCloud">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save cloud</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
